laravel-scout-extended/README.md

```diff
@@ -0,0 +1,71 @@
+#About Algolia's Scout-Extended
+
+Algolia Scout-extended was created bt and mainteained by [Algolia](https://github.com/algolia). The Algolia Search API Client for Laravel. 
+
+##Get started locally
+
+>requires :
+* [Composer](https://getcomposer.org/)
+* [Laravel](https://laravel.com/)
+
+
+
+First, follow Laravel's tutorial to install it. 
+
+>[Laravel tutorial](https://laravel.com/docs/5.8/installation) 
+
+Then: 
+
+```
+# Install from Algolia's Playground from Github
+git clone https://github.com/algolia/api-clients-playground
+
+# Move to Laravel-scout-extended folder.
+cd laravel-scout-extended 
+
+# Install dependencies
+composer install
+
+# Copy .env file as .env.example
+cp .env.example .env
+
+# Add your credential in .env
+ALGOLIA_APP_ID='YOUR APPLICATION ID'
+ALGOLIA_SECRET='YOUR ADMIN API KEY'
+
+# Move to database directory and create database.sqlite
+cd database && touch database.sqlite
```

Comment:
Can't you add a post-composer script for this?

Reply:
And for the migrate, and seed.

Reply:
And the import

Reply:
I can't for the import because the user need to copy past his credencial 
but i can for migrate and seed 👍